src/deque.jl

```diff
@@ -1,3 +1,18 @@
+"""
+    push(vec::StaticVector, item) -> vec
+
+Insert an `item` at the end of `vec` and return the resulting vector.
```

Comment:
"Insert" makes it sound a bit like it is mutating. Maybe something like "Create a new vector with `item` in the end" or similar? (Same on the other docstrings.)